query-engine/driver-adapters/executor/package.json

```diff
@@ -23,19 +23,19 @@
   "license": "Apache-2.0",
   "dependencies": {
     "query-engine-wasm-latest": "npm:@prisma/query-engine-wasm@latest",
-    "query-engine-wasm-baseline": "npm:@prisma/query-engine-wasm@0.0.19",
+    "query-engine-wasm-baseline": "npm:@prisma/query-engine-wasm@0.6.1",
```

Comment:
@prisma/orm-rust should this be updated?

Reply:
No. We can decide to treat a different version as baseline if at some point we think using this version as the baseline is not the right choice anymore, but it definitely must not be updated when just upgrading dependencies.